lib/active_merchant/billing/gateways/digital_river.rb

```diff
@@ -138,6 +139,15 @@ def get_charge_capture_id(order_id)
         )
       end
 
+      def supports_scrubbing?
+        true
+      end
+
+      def scrub(transcript)
+        transcript
+          .gsub(%r((Authorization: Bearer )\w+)i, '\1[FILTERED]\2')
+      end
```

Comment:
This is strange, I added those methods when I was working on adding support for raw gateway logs 🤔

Reply:
Yes indeed 🤔  Maybe it's beacause I moved the get_charge_capture_id method up and it got confused?